### PR TITLE
AUTO-87  Remove keeper turn taking flag

### PR DESCRIPTION
**Pull request by cmalec**

### CODE REVIEW

**Comment by jmank88** [comment]
Should this be mentioned in the changelog?

For TOML config, unrecognized fields will be rejected as invalid, so it would be important for node ops to be aware and reconfigure pre-emptively. Alternatively, we could leave the field in the struct for compatibility, even if it is not actually used (and maybe log a warning).

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -53,7 +53,9 @@ Secrets must be configured manually and passed via `-secrets <filename>` or equi
 
 ### Updated
 
+- OCR2 jobs may now re-use the same contract address on multiple chains.
```
Comment (jmank88) [inline docs/CHANGELOG.md:56]:
```suggestion
```

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -53,7 +53,9 @@ Secrets must be configured manually and passed via `-secrets <filename>` or equi
...
 - `NODE_NO_NEW_HEADS_THRESHOLD=0` no longer requires `NODE_SELECTION_MODE=RoundRobin`. 
+- Removed `KEEPER_TURN_FLAG_ENABLED` as all networks/nodes have switched this to `true` now. The variable should be completely removed my NOPs.
```
Comment (jmank88) [inline docs/CHANGELOG.md:58]: Maybe move this up top now so it doesn't have to be picked back out when 1.11 is merged back to develop

Comment (cmalec) [reply docs/CHANGELOG.md]: moved

**Review by jmank88 — APPROVED** [review]

**Review by infiloop2 — APPROVED** [review]

**Comment by infiloop2** [comment]
Just to confirm @jmank88 is it safe to remove an envvar / config in this way? If a NOP still has the env var / TOML config set, will it throw an error or silently ignore?

**Comment by jmank88** [comment]
> Just to confirm @jmank88 is it safe to remove an envvar / config in this way? If a NOP still has the env var / TOML config set, will it throw an error or silently ignore?

The env var will just be silently ignored.

TOML will reject the unrecognized field, and fail to even parse the file.

**Pull request merged**